src/javis_viewer.jl

```diff
@@ -236,3 +235,15 @@ function _javis_viewer(
     end
 
 end
+
+"""
+     not_repl(video::Video, frames::Int, actions::Vector)
+
+Creates an interactive viewer in a Jupyter Notebook.
+"""
+
+function not_repl(video::Video, frames::Int, objects::Vector)
+    Interact.@manipulate throttle = .35 for f in 1:frames
```

Comment:
Seems like Interact starts with the middle frame which can destroy Javis for certain things like `morph` and `follow_path`. Do you mind setting the slider to the first frame?

Reply:
Yeah. I have experienced this issue. I am looking into how we can fix that.

Reply:
Also, instead of having to use the slider all the time, also have a textbox off to the right of the slider that one could request specific frames for similar to the default viewer would be an asset and achieve feature parity between the different available viewers. Perhaps this would be of assistance: https://juliagizmos.github.io/Interact.jl/latest/widgets/#Text-input-1 and this: https://juliagizmos.github.io/Interact.jl/latest/modifiers/